src/snapred/backend/dao/normalization/NormalizationRecord.py

```diff
@@ -17,25 +17,24 @@ class NormalizationRecord(BaseModel):
 
     """
 
-    runNumber: str
-    useLiteMode: bool
+    # inherits from Record
+    # - runNumber
+    # - useLiteMode
+    # - version
+    # override this to point at the correct daughter class
+    calculationParameters: Normalization
+
+    # specific to normalization records
     backgroundRunNumber: str
     smoothingParameter: float
     peakIntensityThreshold: float
     # detectorPeaks: List[DetectorPeak] # TODO: need to save this for reference during reduction
-    calibration: Calibration
     workspaceNames: List[WorkspaceName] = []
-    version: int = Config["instrument.startingVersionNumber"]
+
     dMin: float
 
```

Comment:
`calibration` is still here, it has jsut been renamed to `calculationParameters`.  The calibration used during normalization can be accessed by its version through `calibrationVersionUsed`

Reply:
What denotes whether something lives on the `NormalizationRecord` vs  its `calculationParameters`?
I would think inputs would go on the parameters (i.e. peakIntensityThreshold), and outputs would either live on the record or be wrapped in another datatype.

This comment is more of a philosophical question rather than a change request, though it may result in a TODO